app/src/main/java/com/kickstarter/ui/viewholders/HorizontalRewardViewHolder.kt

```diff
@@ -35,6 +40,12 @@ class HorizontalRewardViewHolder(private val view: View, val delegate: Delegate)
 
     init {
 
+        val rewardItemAdapter = RewardsItemAdapter()
+        val itemDivider: Drawable = context().getDrawable(R.drawable.divider_grey_500_horizontal)
+        view.horizontal_rewards_item_recycler_view.adapter = rewardItemAdapter
```

Comment:
we can replace the 3 `view.horizontal_rewards_item_recycler_view.` with 
`val itemRecyclerView = view.horizontal_rewards_item_recycler_view.`